bosh-init/bosh_init_cache/minimal-state.json

```diff
@@ -0,0 +1,3 @@
+{
+  "installation_id": "44f01911-a47a-4a24-6ca3-a3109b33f058"
```

Comment:
This was just an arbitrary value taken from one both-init run. I've updated the commit message accordingly.